<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkin_window</class>
 <widget class="QWidget" name="checkin_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>戻る</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>641</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返却</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="user_name_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>所蔵ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckin">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返却</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>this is status label</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="item_identifier">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
